src/stories/Library/Modals/modal-find-on-shelf/modal-find-on-shelf.scss

```diff
@@ -4,11 +4,11 @@
   align-items: flex-start;
 
   @include breakpoint-m {
-    padding: 0 215px $s-4xl 215px;
+    padding: $s-4xl $s-7xl;
   }
 
   &__headline {
-    margin: 72px $s-md 0 $s-md;
+    margin: 0 $s-md;
 
     @include breakpoint-m {
       margin-top: $s-4xl;
```

Comment:
We would need to remove this margin-top here then since we introduce a padding to the modal.
I would suggest always running design system locally and comparing with Figma when doing changes like this so that we don't oversee stuff.

![image](https://user-images.githubusercontent.com/28546954/191012714-1830cc74-76d2-4b81-b722-5902b2df0577.png)